routers/user/home.go

```diff
@@ -307,10 +308,23 @@ func Issues(ctx *context.Context) {
 		return
 	}
 
+	var issuesStates = make([]*models.CommitStatus, 0, len(issues))
 	for _, issue := range issues {
 		issue.Repo = showReposMap[issue.RepoID]
+
+		if issue.IsPull {
+			issue.LoadAttributes()
```

Comment:
Let's not ignore the error returned here

Reply:
done.